### PR TITLE
Topic recognition(ISICs Segmentation) merge request

### PR DESCRIPTION
**Pull request by sleepy-duo**
**Student Name: Xin Qi.    Student ID: S45104208**
This pull request is to merge the task4: Segment the ISICs dataset into the PatternFlow library.

There are two files: model.py and driver.py in the folder "Task4_ISIC_seg_S4510420_Xin_Qi". The model.py builds a class that includes the building improved U-net model and some useful built-in functions. The driver.py demonstrates how to import the model.py and the output meaningful results through built-in functions.

If you encounter some problems when merging this pull request, feel free to give me feedback.
Thank you.

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Sufficiently detailed commit messages
- Code valid for Normal Difficulty
- Results are good (dsc > 0.8)

Other Feedback
- Leave .DS_Store out of your commit using .gitignore

**Comment by Xin-Qi** [comment]
> * Sufficiently detailed commit messages
> * Code valid for Normal Difficulty
> * Results are good (dsc > 0.8)
> 
> Other Feedback
> 
> * Leave .DS_Store out of your commit using .gitignore

--Update--
Already remove the extra .DS_Store file and create .gitignore file.

**Comment by rexsw** [comment]
Code looks good for normal difficulty

I have no other feedback, good work!

File: recognition/Task4_ISIC_seg_S4510420_Xin_Qi/driver.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
+__author__ = "Xin Qi"
```
Comment (shakes76) [inline recognition/Task4_ISIC_seg_S4510420_Xin_Qi/driver.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

**Pull request merged**